File: src/BooKing.Identity/BooKing.Identity.Application/Dtos/ReturnLoginUserDto.cs
namespace BooKing.Identity.Application.Dtos;
public class ReturnLoginUserDto
{
    public string AccessToken { get; set; }
    public double ExpiresIn { get; set; }    

    public Guid UserId { get; set; }
    public string UserEmail { get; set; }
    public string UserName { get; set; }
}

File: src/BooKing.Web/src/app/dtos/returnloginuser.dto.ts
export class ReturnLoginUserDto {
  accessToken: string;
  expiresIn: number;
  userId: string;
  userEmail: string;
  userName: string;

  constructor(
    accessToken: string,
    expiresIn: number,
    userId: string,
    userEmail: string,
    userName: string
  ) {
    this.accessToken = accessToken;
    this.expiresIn = expiresIn;
    this.userId = userId;
    this.userEmail = userEmail;
    this.userName =  userName;
  }
}
